mne/commands/utils.py

```diff
@@ -4,14 +4,29 @@
 #
 # License: BSD (3-clause)
 
-import imp
+import sys
 import os
 import re
 from optparse import OptionParser
 
 import mne
 
 
+def load_module(name, path):
+    if sys.version_info < (3, 3):
+        import imp
+        return imp.load_source(name, path)
```

Comment:
Agreed, now I only have to find a way to do this for Python 3.3/3.4. I can use `imp.load_compiled` for Python < 3.3, and the approach for 3.5+ also works for .pyc files. Any ideas/hints?